tests/tensorflow_autolog/test_tensorflow2_autolog.py

```diff
@@ -53,6 +54,26 @@ def manual_run(request):
     mlflow.end_run()
 
 
+@pytest.fixture
+def clear_tf_keras_imports():
+    """
+    Simulates a state where `tensorflow` and `keras` are not imported by removing these
+    libraries from the `sys.modules` dictionary. This is useful for testing the interaction
+    between TensorFlow / Keras and the fluent `mlflow.autolog()` API because it will cause import
+    hooks to be re-triggered upon re-import after `mlflow.autolog()` is enabled.
+    """
+    sys.modules.pop("tensorflow", None)
+    sys.modules.pop("keras", None)
+
+
+@pytest.fixture(autouse=True)
+def clear_fluent_autologging_import_hooks():
+    """
+    """
```

Comment:
```suggestion
```

Can we remove this?

Reply:
Added a docstring here! Thanks for catching this :)